### DISCUSSION TITLE
小测试, 发现reality不会真的去访问 "serverNames":

### DISCUSSION THREAD

**Discussion by sharklxw** [XTLS/Xray-core#2691] [General]
本人vpsA使用瓦工，换机房得到一个新的ip，然后搭建reality，"dest":到一个国外可用的网站，"serverNames":写自己vpsB搭建的网站。
使用reality看油管，并查看vpsB上的网站访问记录，没有发现vpsA的ip。
如果更改"dest":，写自己vpsB的网站，就可以看到vpsA的访问记录了。
.
这是否就是reality可以无sni，任意sni的原因？当然我是小白。我最想问，如果对reality进行抓包，或者通过GFW，究竟是可以看到"dest"的网站，还是只能看到"serverNames"的网站？ 本人还没有学会抓包https,,,也没有看到有人抓包reality，，

**Comment by chika0801** [comment]
我认为可参考 https://github.com/XTLS/Xray-core/issues/1681#issuecomment-1436655742

如果看不懂，不必强行让自己懂。会使用方法就行了。

https://github.com/XTLS/Xray-core/issues/1681#issuecomment-1436660954

**Comment by chise0713** [comment]
访问的是dest而servername只是sni
